test/evptests.txt

```diff
@@ -2,6 +2,63 @@
 #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
 #digest:::input:output
 
+# BLAKE2 tests, using same inputs as MD5
```

Comment:
How were these generated? Are these official / do we have any official test vectors?